ocaml/xapi/xapi_globs.ml

```diff
@@ -681,13 +681,13 @@ let ha_xapi_restart_timeout = ref 300
    logrotate when it exceeds the threshold *)
 let logrotate_check_interval = ref 300.
 
-let rrd_backup_interval = ref 86400.
+let rrd_backup_interval = ref 86400
 
 (* CP-703: Periodic revalidation of externally-authenticated sessions *)
-let session_revalidation_interval = ref 300. (* every 5 minutes *)
+let session_revalidation_interval = ref 300 (* every 5 minutes *)
```

Comment:
The reason that we can't express these in spans is that these refs are updated by command-line options?

Reply:
Indeed, I'm not sure these bindings can be changed to integers without breaking anything.

In theory we could deserialize Spans in the form of an integer followed by the unit of time.